.circleci/config.yml

```diff
@@ -30,7 +30,7 @@ jobs:
 
           if [ $CURL_FAILED -eq 1 ]
           then
-            npm ci && cd node_modules/mattermost-redux && npm install && npm run build && cd ../.. && make build
+            npm ci && make build
```

Comment:
Same question here: Why don't we need to build this any longer?

Reply:
I am not entirely sure that this step was really needed before 🤔

Reply:
https://github.com/mattermost/mattermost-webapp/pull/4064#issuecomment-546695748